json_ai_files/save_load_details.json

```diff
@@ -102,7 +102,7 @@
             "False": {
                 "save_num": "25",
                 "blitz_win_value": "True",
-                "end_game_version_val": "1",

```

Comment:
Do we want to add these changes in too? Without the saves it's not likely going to result in checkpoints that work.

Reply:
This part shouldn't matter, considering how it's before the end_game_version_val would be used (it's related to B&Y drops)